src/main/java/Bob.java

```diff
@@ -31,7 +31,7 @@ else if (input.equals("list")) {
                 }
             }
 
-            else if (input.trim().matches("mark|unmark|deadline|todo|event")) {
+            else if (input.trim().matches("mark|unmark|deadline|todo|event|delete")) {
```

Comment:
Small nit. I believe the else if should be preceded by the closing curly brackets from the if/if else statement above it.